doc/manual/en/install/generic.rst

```diff
@@ -23,21 +23,101 @@ These are
     the prerequisite is PHP 7. The core of the CometVisu itself can be used
     without PHP though.
 
-Backend - knxd/eibd or OpenHAB
---------------------------------
+Backend
+-------
 
-As a backend an instance of *eibd* or its successor
-`knxd <https://github.com/knxd/knxd>`__ can be used as well an
-`OpenHAB <https://www.openhab.org/>`__ installation.
+To be able to connect the devices with the CometVisu you need a "backend".
+There are different possibilities:
 
+knxd / eibd
+~~~~~~~~~~~
+
+For a direct connection to the KNX bus the knxd is used.
 The installation of the backend is described under
-:doc:`backends/install-eibd` .
+:doc:`backends/install-eibd`.
 
 .. toctree::
     :hidden:
 
     backends/install-eibd
 
+openHAB
+~~~~~~~
+
+To use openHAB as backend you can refer to
+:doc:`openhab`
+and
+:doc:`docker_openhab`.
+
+MQTT
+~~~~
+
+To use the CometVisu as one of the MQTT participant you need a MQTT broker
+that can be contaced with WebSockets.
+
+Backend configuration
+~~~~~~~~~~~~~~~~~~~~~
+
+To configure the CometVisu to use a given backend different possibilities are
+available, listed below.
+There a later listed method overrules those above it, e.g. the values from the
+config file will be used even when those are set differently with docker
+environment variables.
+
+HTTP header / Docker environment variable
+.........................................
+
+The web server can pass setup information in the HTTP header of the
+config file. By using the official docker image of the CometVisu this can
+be achieved by setting the corresponding ``ENVIRONMENT`` parameters.
+
+============================ ================ ===========
+HTTP header                  ``ENVIRONMENT``  use
+---------------------------- ---------------- -----------
+X-CometVisu-Backend-Name     BACKEND_NAME     Name like ``knxd``, ``oh`` or ``mqtt``
+X-CometVisu-Backend-Url      BACKEND_URL      URL of the MQTT WebSocket
+X-CometVisu-Backend-LoginUrl CGI_URL_PATH     URL of the knxd or openHAB login ressource
+X-CometVisu-Backend-User     BACKEND_USERNAME User name, when needed for the MQTT broker
+X-CometVisu-Backend-Pass     BACKEND_PASSWORD Password, when needed for the MQTT broker
+============================ ================ ===========
+
+.. warning::
+
+    The user name and password for the MQTT broker are stored in pain text
+    and are transported like that over the network!
+
+Config file
+...........
+
+In the all containing ``<pages>`` element the different parameters can be set
+as attributes:
+
+=========== ===========
+attribute   use
+----------- -----------
+backend     Name like ``knxd``, ``oh`` or ``mqtt``
```

Comment:
we should go with "openhab" here, all the other variants are currently kept for backwards compability and should be removes with the next library upgrade